src/main/scala/com.snowplowanalytics.snowplow.micro/Main.scala

```diff
@@ -20,7 +20,7 @@ object Main
     header =
       s"""Notes:
          |    - With --storage, /micro/all, /micro/good and /micro/bad endpoints are not supported.
-         |    - With --max-events set to 0, all /micro/* API endpoints are disabled.""".stripMargin,
+         |    - With --no-storage all /micro/* API endpoints are disabled.""".stripMargin,
```

Comment:
I think we need to rename `--storage` to `--storage-file`, otherwise this message is weird. It looks like `--storage` and `--no-storage` are the two alternative settings and both restrict the endpoints.